lahja/endpoint.py

```diff
@@ -467,21 +451,13 @@ def _throw_if_already_connected(self, *endpoints: ConnectionConfig) -> None:
         self._receiving_loop_running.set()
         while self._running:
             (item, config) = await self._receiving_queue.get()
+            self.logger.info('%s: PROCESSING ITEM FROM RECEIVING QUEUE: %s', self.name, item)
             try:
                 event = self._decompress_event(item)
                 self._process_item(event, config)
             except Exception:
                 traceback.print_exc()
 
-    async def _connect_internal_queue(self) -> None:
```

Comment:
@cburgdorf thoughts on this change.  Seems like a nice simplification to just process internal events immediately.